src/stories/pages/Home/Home.js

```diff
@@ -113,7 +135,8 @@ export const Home = () => {
           popularRecipes.map((recipe) => {
             return ({
               id: recipe.id,
-              isFavorite: false,
+              isFavorite: profile &&
+                profile.favoriteRecipes.includes(recipe.id),
               url: `/recipes/${recipe.id}`,
               image: `https://spoonacular.com/recipeImages/${recipe.id}-636x393.jpg`,
```

Comment:
You can probably create a global function/variable for the image source. DRYer code.